source/dotnet/wholesale-api/WebApi/Configuration/Modules/EventsRegistration.cs

```diff
@@ -61,6 +83,9 @@ private static void AddApplication(this IServiceCollection serviceCollection)
         serviceCollection
             .AddScoped<IEnergyResultEventProvider, EnergyResultEventProvider>()
             .AddScoped<IWholesaleResultEventProvider, WholesaleResultEventProvider>();
+
+        serviceCollection.AddScoped<IGridAreaOwnerRepository, GridAreaOwnerRepository>();
+        serviceCollection.AddScoped<IReceivedIntegrationEventRepository, ReceivedIntegrationEventRepository>();
```

Comment:
Use the same formatting as other code within the same method

```suggestion
        serviceCollection
            .AddScoped<IGridAreaOwnerRepository, GridAreaOwnerRepository>()
            .AddScoped<IReceivedIntegrationEventRepository, ReceivedIntegrationEventRepository>();
```